www/proxy-sso/src/etc/inc/plugins.inc.d/proxy_sso.inc

```diff
@@ -0,0 +1,13 @@
+<?php
+
+function proxy_sso_configure()
+{
+    return [
+        'webproxy_reconfigure' => ['proxy_sso_squid_reconfigure'],
```

Comment:
does this work in the core or do we need to merge a pr? I vaguely remember something

Reply:
This needs https://github.com/opnsense/core/pull/1627

Reply:
this changes slightly with the merge: the facility is "webproxy" and in reconfigure you need to check for "reconfigure" argument, it looks like this:

https://github.com/opnsense/core/blob/master/src/etc/inc/plugins.inc.d/openvpn.inc#L37

Reply:
Some changes have to be done in connection with acceptance of https://github.com/opnsense/core/pull/1627

Reply:
Done: https://github.com/opnsense/plugins/pull/266/commits/651ce706ac2f785cc3a1d4b022e752800dec8f79

Reply:
do we even need proxy_sso_configure then?

Reply:
yes

Reply:
in that case the line now reads something like:

'webproxy' => ['proxy_sso_squid_reconfigure:2'],

but the second argument to the function is the "reconfigure" statement like

https://github.com/opnsense/core/blob/master/src/etc/inc/plugins.inc.d/openvpn.inc#L1172

Reply:
Why? I don't understand why I need set count of parameters to 2 when function proxy_sso_squid_reconfigure() has no parameters.

Reply:
because the plugin configure endpoint was simplified. we can pass all kinds of arguments here, we don't have to encode the type of the call in the name of the endpoint, so "webproxy_reconfigure" becomes "webproxy":

https://github.com/opnsense/core/commit/b769f359b3ea2c7cf49ca87495d0b2c7464abb68#diff-7ab5c46a2101377cc2113c978d5a9676R34

Reply:
aha, understood )

Reply:
done!